src/Pages/Authentication/Login.jsx

```diff
@@ -0,0 +1,87 @@
+import axios from "axios";
+import React,{ useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+import "./Auth.css";
+
+export const Login = () => {
+  const navigate = useNavigate();
+
+  const [credentials, setCredentials] = useState({ email: "", pass: "" });
+
+  const dummyUser = {
+    email: "adarshbalika@gmail.com",
+    pass: "adarshBalika123",
+  };
+
+  const addDummyUser = (e) => {
+    e.preventDefault();
+    setCredentials(dummyUser);
+  };
+  
+  const dataHandler = (e) => {
+    let name = e.target.name;
+    let value = e.target.value;
+    setCredentials({ ...credentials, [name]: value });
```

Comment:
You can mdify the declaration of fields by this way.
`const {target:{ name, value } = e `